examples/advanced/denver-trees.html

```diff
@@ -41,11 +41,16 @@ <h1>Denver trees</h1>
       style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
       center: [-105.048, 39.749],
       zoom: 15,
-      dragRotate: false
+      scrollZoom: false,
+      dragRotate: false,
+      touchZoomRotate: false,
     });
 
+<<<<<<< HEAD
```

Comment:
Nice merge :P

Reply:
:face_with_head_bandage: I did the merge fine... but I didn't save the files... committing now